samples/dnn/vit_tracker.cpp

```diff
@@ -0,0 +1,176 @@
+// VitTracker
+// model: https://github.com/opencv/opencv_extra/blob/4.x/testdata/dnn/onnx/models/vitTracker.onnx
```

Comment:
Use opencv_zoo too to avoid mess of different sources

Also it makes sense to dump suggested model path in `--help` mode.